util/build/enclave/Cargo.toml

```diff
@@ -11,8 +11,8 @@ mc-util-build-script = { path = "../script" }
 mc-util-build-sgx = { path = "../sgx" }
 mc-sgx-css = { path = "../../../sgx/css" }
 
-mbedtls = { git = "https://github.com/mobilecoinofficial/rust-mbedtls.git", tag = "mc-0.3", default-features = false }
-mbedtls-sys-auto = { git = "https://github.com/mobilecoinofficial/rust-mbedtls.git", tag = "mc-0.3", default-features = false }
+mbedtls = { version = "0.8.1", default-features = false, features = ["no_std_deps"] }
```

Comment:
Now that we're doing `resolver = "2"`, this can probably just be `mbedtls = "0.8.1"`